graylog2-server/src/main/java/org/graylog2/users/UserOverviewDTO.java

```diff
@@ -107,6 +108,10 @@
     @JsonProperty(FIELD_ACCOUNT_STATUS)
     public abstract User.AccountStatus accountStatus();
 
+    @Nullable
+    @JsonProperty(FIELD_SERVICE_ACCOUNT)
+    public abstract Boolean serviceAccount();
```

Comment:
Let's make this `boolean` and remove the `@Nullable` annotation. That makes the usage of the property easier because consumers don't have to do `null` checks.